### ISSUE TITLE
Expected to have finished reduction in the prior iteration before starting a new one. This error indicates that your module has parameters that were not used in producing loss.

### ISSUE THREAD

**Issue by starsky68** [Megvii-BaseDetection/YOLOX#885]
Expected to have finished reduction in the prior iteration before starting a new one. This error indicates that your module has parameters that were not used in producing loss. You can enable unused parameter detection by (1) passing the keyword argument `find_unused_parameters=True` to `torch.nn.parallel.DistributedDataParallel`; (2) making sure all `forward` function outputs participate in calculating loss. If you already have done the above two steps, then the distributed data parallel module wasn't able to locate the output tensors in the return value of your module's `forward` function. Please include the loss function and the structure of the return value of `forward` of your module when reporting this issue (e.g. list, dict, iterable).

**Comment by LiXiang0021** [comment]
have you solved it??

**Comment by VCBE123** [comment]
> have you solved it??

@LiXiang0021 
I meet the same problem when I use the ConvNext backbone.
And in the forward function of PAFPN
```python
features = self.backbone(input)
[x2,x1,x0] = features
```
Change these two line to 
```python
[x2,x1,x0] = self.backbone(input)
```
Then the problem was solved. It may because the **features** is unused parameter in forward pass.
